.github/workflows/ci.yml

```diff
@@ -11,9 +11,13 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
     - name: Install dependencies
-      run: yarn && yarn add -D eslint-config-airbnb-base eslint-plugin-unicorn eslint-plugin-jest@^26.9.0 @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript
+      run: yarn && yarn add -D eslint-config-airbnb-base eslint-plugin-unicorn eslint-plugin-jest@^26.9.0 @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript github:fomantic/prettier#ca4fc1130f
     - name: Lint JS and LESS files
-      run: npm run lint
+      run: >
+        npm run lint
```

Comment:
I suggest to move the prettier check into a separate task instead of being part of the linter

Reply:
done

all feedback should be addressed now

Reply:
done

all feedback should be addressed now